ext/web/09_file.js

```diff
@@ -320,8 +320,6 @@
       webidl.assertBranded(this, BlobPrototype);
       const partIterator = toIterator(this[_parts]);
       const stream = new ReadableStream({
-        type: "bytes",
-        /** @param {ReadableByteStreamController} controller */
```

Comment:
(comment removed because removing `type: "bytes"` changes the type of `controller` to be ReadableStreamDefaultController instead which is inferred fine, and not specified in other similar bits of code)